core-feature-pack/src/main/resources/content/docs/contrib/scripts/systemd/wildfly.service

```diff
@@ -5,12 +5,15 @@ Before=httpd.service
 
 [Service]
 Environment=LAUNCH_JBOSS_IN_BACKGROUND=1
+EnvironmentFile=-/etc/wildfly/wildfly.d/*
 EnvironmentFile=-/etc/wildfly/wildfly.conf
+Sockets=httpd.socket
```

Comment:
But 80 is (at least in Fedora: https://pkgs.fedoraproject.org/cgit/rpms/httpd.git/ )

The main reason why 8080 is used by default on wildfly and elsewhere is because of the kernel limitation on port numbers. Using systemd, this is no longer a problem.